ui/src/shared/utils/getOrgIDFromUrl.test.ts

```diff
@@ -0,0 +1,39 @@
+import {getOrgIDFromUrl} from 'src/shared/utils/getOrgIDFromUrl'
+
+const baseUrl = 'http://influx.application/'
+const orgID = 'f440096db5cdb0b1'
+const pathname = `/orgs/${orgID}`
+const location = {
+  ...window.location,
+  href: `${baseUrl}${pathname}`,
+  pathname,
+}
+Object.defineProperty(window, 'location', {
```

Comment:
why are you over loading a host variable? there be dragons there, and is not recommended outside of w3c ordained polyfills.

Reply:
oh.. for tests... hmmmmm